src/pages/Home/PathwayCard.js

```diff
@@ -4,45 +4,40 @@ import { Link } from "react-router-dom";
 import { PATHS, interpolatePath } from "../../constant";
 import { Typography, CardMedia, CardContent, Card, Box } from "@mui/material";
 
-function PathwayCard({ id, title, description, image, hover }) {
+function PathwayCard({ id, name, description, logo, hover }) {
   const classes = useStyles();
 
   return (
     <Link
       to={
         id
           ? interpolatePath(PATHS.PATHWAY_COURSE, { pathwayId: id })
-          : title === "Miscellaneous Courses"
-          ? PATHS.MISCELLANEOUS_COURSE
-          : title === "Residential Programmes" && PATHS.RESIDENTIAL_COURSE
+          : PATHS.MISCELLANEOUS_COURSE
       }
       className={classes.link}
-      style={{ pointerEvents: hover === false && "none" }}
-    >
+      style={{ pointerEvents: hover === false && "none" }}>
       {hover ? (
         <Card
           elevation={2}
-          className={hover ? classes.card : image && classes.imageCard}
-        >
-          {image && (
+          className={hover ? classes.card : logo && classes.imageCard}>
+          {logo && (
             <CardMedia
               component="img"
               src={
-                image.includes('https')
-                  ? image
-                  : require("./assets/" + image + ".svg")
+                logo.includes("https")
+                  ? logo
+                  : require("./assets/" + logo + ".svg")
```

Comment:
May want to have a function that returns the paths of the logos to use based on what comes from the back-end. This would allow for changes (e.g., if logos aren't going to be in assets folder) without editing the logic in multiple components.